e2e/express-ts/package.json

```diff
@@ -8,7 +8,8 @@
 		"build": "tsc",
 		"dev": "ts-node src/index.ts",
 		"start": "node dist/index.js",
-		"test:dev": "ts-node src/test.ts"
+		"test:dev": "ts-node src/test.ts",
+		"sourcemaps": "yarn exec ../../sourcemap-uploader/dist/index.js upload --apiKey \"${HIGHLIGHT_API_KEY}\" --appVersion \"${APP_VERSION}\" --path ./dist --basePath /Users/vkorolik/work/highlight/e2e/express-ts/dist/"
```

Comment:
Not a blocker, but this will only work for your laptop with path `/Users/vkorolik/work`?

Reply:
good point, making this the absolute working dir to make this work for others